xsuite/src/data/index.test.ts

```diff
@@ -842,6 +842,52 @@ test("e.account", async () => {
   );
 });
 
+test("e.MapperKey - simple key", () => {
+  expect(e.MapperKey("key").toTopHex()).toEqual("6b6579");
+  expect(e.MapperKey("key").toNestHex()).toEqual("6b6579");
+});
+
+test("e.MapperKey - complex key", () => {
+  expect(e.MapperKey("key", e.U32(1)).toTopHex()).toEqual("6b657900000001");
+  expect(e.MapperKey("key", e.U32(1)).toNestHex()).toEqual("6b657900000001");
+});
+
+test("e.EsdtRolesKey", () => {
```

Comment:
In case "Roles" is changed into "Role", it should be changed here. To be disregarded if we don't change here.

Reply:
👍